iDiff/tests/fileDiff_test_processor.py

```diff
@@ -27,7 +27,7 @@ def _trim_file_names(files):
 
 
 def _trim_layer_hash(filename):
-    hash_match = re.match(r'^([a-z|0-9]{64})/', filename)
+    hash_match = re.match(r'^([a-z|0-9])/', filename)
```

Comment:
not sure why this was changed... but anyways, theoretically, we wouldn't even need these post-processing scripts anymore now that we have the final filesystem and no layer folders, right?